easybuild/toolchains/pgi.py

```diff
@@ -0,0 +1,36 @@
+##
+# Copyright 2015 Bart Oldeman
```

Comment:
transfer copyright to Ghent University, or see https://github.com/hpcugent/easybuild-framework/blob/master/CONTRIBUTING.md